src/types/register_request.rs

```diff
@@ -21,9 +19,69 @@ pub enum RegisterRequest {
 }
 
 #[derive(Debug, Serialize, Deserialize, Clone, JsonSchema)]
+/// Parameters of registered RTP input stream.
+/// Before using input in video composition or output mixing,
+/// input has to be firstly registered using `register_input` request.
+///
+/// At least one of `video` and `audio` has to be defined.
 pub struct RegisterInputRequest {
     pub input_id: InputId,
+    /// Port on which RTP stream is send.
     pub port: Port,
+    /// Represents video received on input.
+    /// If set, `input_id` stream can be used in video composition.
```

Comment:
```suggestion
    /// Parameters of a video source included in the RTP stream.
```

> If set, `input_id` stream can be used in audio mixing.

It should be valid to use stream without video for video mixing and without audio for audio mixing.


I used "video source" because stream would sound weird if I use "RTP stream" in the same sentence. Also s in SSRC stands for source so it seems fitting